README.md

```diff
@@ -171,7 +162,7 @@ with nGraph backends.
 Note: The actual filename for the pip package may be different as it's version 
 dependent. Please check the `build/python/dist` directory for the actual pip wheel.
 
```

Comment:
Okay, I rewrote this idea a bit since it's not really "Option 3" and more of an alternative to building from source.  The explanation makes more sense when that detail is added so thanks!